### PR TITLE
Ticket skeleton

### PR DESCRIPTION
**Pull request by jpechane**
Skeleton order service, defines a development workflow

### CODE REVIEW

File: ticket-msa/invoice/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Invoice Microservice
+Pre-requisities
+* Kafka broker service
+
+
+## Deploy service to OpenShift from source
+
+```
+$ oc new-app --name=invoice-msa fabric8/s2i-java:latest~https://github.com/jpechane/strimzi-lab#ticket-skeleton \
```
Comment (gunnarmorling) [inline ticket-msa/invoice/README.md]: Can you change it to the upstream repo?

Comment (jpechane) [reply ticket-msa/invoice/README.md]: Thanks for spotting, is is a leftover from testing

File: ticket-msa/order/.gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+target
```
Comment (gunnarmorling) [inline ticket-msa/order/.gitignore]: I'll remove that.

File: ticket-msa/invoice/src/main/resources/project-defaults.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+swarm:
```
Comment (gunnarmorling) [inline ticket-msa/invoice/src/main/resources/project-defaults.yml]: Where/how is this file consumed?

**Comment by gunnarmorling** [comment]
Applied. Leaving it open only to get closure on that yml file.

File: ticket-msa/invoice/src/main/resources/project-defaults.yml (hunk 3 above)
Comment (jpechane) [reply ticket-msa/invoice/src/main/resources/project-defaults.yml]: Is is Swarm/Thorntail config file - see https://docs.thorntail.io/2.2.0.Final/#default-thorntail-yaml-files_thorntail

Comment (gunnarmorling) [reply ticket-msa/invoice/src/main/resources/project-defaults.yml]: Ah, got it. What happened to good ol' _standalone.xml_. ;)

**Pull request closed without merging**